easybuild/easyconfigs/i/InParanoid/InParanoid-5.0.79d8e39-GCC-10.3.0.eb

```diff
@@ -0,0 +1,61 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+easyblock = 'Tarball'
+
+name = 'InParanoid'
+version = '5.0.79d8e39'
```

Comment:
@smoretti I only notice now that this is a self-invented version...

I think it would be better to ensure we use a version that sorts well, so use a datestamp of the commit instead?

```python
version = '5.0-20220607`